docs/interfaces.md

```diff
@@ -460,6 +460,13 @@ processes on the system.
 
 * Auto-Connect: no
 
+### system-power-control
```

Comment:
Can we please call this one "reboot-control"?  This seems shorter and more clearly associated with this action.